portfolio/src/main/java/com/google/sps/servlets/ContactInformationServlet.java

```diff
@@ -26,11 +33,37 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     boolean professionalContact = Boolean.parseBoolean(getParameter(request, "professional-contact", "false"));
     boolean otherContact = Boolean.parseBoolean(getParameter(request, "other-contact", "false"));
     String otherContactReason = getParameter(request, "other-contact-reason", "");
+    String contactReason = "";
 
     System.out.println("New contact: " + firstName + " " + lastName + ", " + email);
-    if (academicContact) System.out.println("Contact reason: academic");
-    if (professionalContact) System.out.println("Contact reason: professional");
-    if (otherContact) System.out.println("Contact reason: other. " + "Reason: " + otherContactReason);
+    if (academicContact) {
```

Comment:
You can also use contactReason += "your String"; or contactReason.concat("your String"); as a way to not write contactReason two times. Other than that, your code looks great! it is really organized (:

Reply:
You're right! I always forget that works for `String`s too and not just numbers.